src/capi.cpp

```diff
@@ -529,6 +529,27 @@ int rtcAddRemoteCandidate(int pc, const char *cand, const char *mid) {
 	});
 }
 
+int rtcGetLocalDescriptionSdp(int pc, char *buffer, int size) {
+	return WRAP({
+		auto peerConnection = getPeerConnection(pc);
+
+		if (size <= 0)
+			return 0;
+
+		if (!buffer)
+			throw std::invalid_argument("Unexpected null pointer for buffer");
+
+		if (auto desc = peerConnection->localDescription()) {
+			auto sdp = string(*desc);
+			const char *data = sdp.data();
+			size = std::min(size - 1, int(sdp.size()));
+			std::copy(data, data + size, buffer);
+			buffer[size] = '\0';
+			return size + 1;
+		}
```

Comment:
I approved a bit too fast, here it would be better to handle the case where `desc` is not present and return `RTC_ERR_FAILURE`, otherwise the wrapper will return `RTC_ERR_SUCCESS` (i.e. 0) by default, which sounds more like the description is there but empty.

Reply:
I added it.

One thing I would like to ask is, does that mean rtcGetLocalAddress and rtcGetRemoteAddress should be modified the same way?

Reply:
Yes indeed, I'll do it!